kalite/topic_tools/__init__.py

```diff
@@ -168,12 +187,18 @@ def get_exercise_cache(force=False, language=None):
         EXERCISES = {}
     if EXERCISES.get(language) is None:
         if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
-            exercises = softload_json(settings.EXERCISES_FILEPATH + "_" + language + ".cache", logger=logging.debug, raises=False)
+            exercises = softload_json(
+                cache_file_path("exercises_{0}.json".format(language)),
+                logger=logging.debug,
+                raises=False
+            )
             if exercises:
                 EXERCISES[language] = exercises
                 return EXERCISES[language]
         EXERCISES[language] = softload_json(settings.EXERCISES_FILEPATH, logger=logging.debug, raises=False)
-        if language == "en":  # English-language exercises live in application space, translations in user space
+        
+        # English-language exercises live in application space, translations in user space
+        if language == "en":
```

Comment:
Purely a matter of style, but I like comments that live on the same line as the thing they are annotating, as it makes them harder to separate/get out of sync. Imagine this
```python
# talking about foo
do_foo()
```

then later something gets added:
```py
# talking about foo
do_bar()
do_foo()
```

Then `do_foo` is decided to be obsolete and replaced with `do_baz`, so:
```python
# talk about foo
do_bar()
do_baz()
```